src/components/AuthorCardEditing.vue

```diff
@@ -207,6 +212,51 @@ export default defineComponent({
             orcidErrors
         }
     },
+    watch: {
+        // eslint-disable-next-line @typescript-eslint/no-unused-vars
+        orcid (_oldVal, _newVal) {
+            if (this.orcid.length === 37 && this.orcidErrors.length === 0) {
+                // If a valid orcid is found, look for data in the orcid API (maybe only if we do not have data already)
+                const orcidSearchApi = 'https://pub.orcid.org/v3.0/expanded-search/?q=orcid:'
+                const orcidEndpoint = this.orcid.replace('https://orcid.org/', orcidSearchApi) + '&rows=1'
+                // This is a test endpoint on Zenodo sandbox.
+                // const orcidEndpoint = 'https://pub.sandbox.orcid.org/v3.0/expanded-search/?q=orcid:0000-0001-8555-849X&rows=1';
+
+                axios.interceptors.request.use((config) => {
+                    config.props = this
+                    return config
+                })
+
```

Comment:
This should fix some of the typescript linter below
```suggestion
                interface OrcidAuthor {
                    'given-names': string
                    'family-names': string
                    email: string
                    'institution-name': string[]
                }
                interface OrcidResponse {
                    'expanded-result': OrcidAuthor[]
                }
```
ref: https://stackoverflow.com/questions/62217642/react-and-typescript-which-types-for-an-axios-response